src/game/Player.cpp

```diff
@@ -2447,14 +2447,12 @@ bool Player::IsGroupVisiblefor (Player* p) const
 
 bool Player::IsInSameGroupWith(Player const* p) const

```

Comment:
This function is used for other things, too, you should not change it like this.